src/main/java/seedu/address/model/AnakinReadOnlyAnakin.java

```diff
@@ -13,16 +13,16 @@
      * Returns an unmodifiable view of the decks list.
      * This list will not contain any duplicate decks.
      */
-    ObservableList<AnakinDeck> getDeckList();
+    ObservableList<Deck> getDeckList();
 
     /**
      * Returns an unmodifiable view of the cards list.
      * This list will not contain any duplicate cards.
      */
     ObservableList<AnakinCard> getCardList();
 
-    /**
-     * Returns the state
-     */
     boolean isInsideDeck();
+     */
+     * Returns the state
```

Comment:
Fix please

Reply:
Yes pls. Also I think we should look at the Travis debug log for this one since by right this PR should solve the compile issues that caused us to turn it off.